pkgs/build-support/setup-hooks/strip.sh

```diff
@@ -62,7 +62,12 @@ stripDirs() {
     if [ -n "${paths}" ]; then
         echo "stripping (with command $cmd and flags $stripFlags) in $paths"
         # Do not strip lib/debug. This is a directory used by setup-hooks/separate-debug-info.sh.
-        find $paths -type f -a '!' -wholename "$prefix/lib/debug/*" -exec $cmd $stripFlags '{}' \; 2>/dev/null
+        find $paths -type f -a '!' -path "$prefix/lib/debug/*" -print0 |
+            xargs -r -0 -n1 -P "$NIX_BUILD_CORES" $cmd $stripFlags 2>/dev/null || exit_code=$?
```

Comment:
@trofi maybe a -- before cmd is needed?

Reply:
Added to my PR, building..

Reply:
Now it fails with `xargs: -s: No such file or directory`

Reply:
Aha there's no $cmd
`stripping (with command  and flags -s) in`
`echo "stripping (with command $cmd and flags $stripFlags) in $paths"`

Reply:
On current master(3ec081d54ebed5f8f308efbc0644f9e65ef297d9) it has the same issue
`extra-utils> stripping (with command  and flags -s) in `

so this issue was just being ignored!

Reply:
https://github.com/NixOS/nixpkgs/pull/244449